test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -332,40 +354,43 @@ export default class PerfCompare extends Component {
         
         // Received a Perffarm Run URL
         } else {
-
-            const [baselineSchemeWithHostWithPort, baselineBuildNum] = this.state.inputURL.baselineID.split("build_id=");
-            const [testSchemeWithHostWithPort, testBuildNum] = this.state.inputURL.testID.split("build_id=");
-
-            // Check if the benchmark and test data is valid
-            if (baselineBuildNum === undefined || (baselineSchemeWithHostWithPort.substring(0,31) !== "http://perffarmServer")) {
-                displayErrorMessage += "Invalid Baseline URL. "
-            }
-            if (testBuildNum === undefined || (testSchemeWithHostWithPort.substring(0,31) !== "http://perffarmServer")) {
-                displayErrorMessage += "Invalid Test URL. "
-            }
-
-            // Data received is not valid
-            if (displayErrorMessage !== "") {
-                this.setState(
-                    {
-                        displayAlert: {
-                            status: true,
-                            message: displayErrorMessage
-                        }
-                    }
-                )
-                return
-            }
-
-            await this.setState(
-                {
-                    selectedRuns: {
-                        baselineID: baselineBuildNum,
-                        testID: testBuildNum,
-                    }
-                }
-            )
-
+        	
+        	
+        	const [baselineSchemeWithHostWithPort, baselineBuildNum] = this.state.inputURL.baselineID.split("build_id=");
```

Comment:
@sophiaxu0424 Why is the indentation not the same as before? We should only change the lines that we need to. Otherwise, it becomes less convenient to review the changes since we see a bigger diff. Make to review all changes including indentations before creating or updating a PR.

Reply:
Fixed it, thank you.